commons/FPath.ml

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
you need the license boilerplate at the top (see file_type.ml top comment for example)